apps/smartling/frontend/src/Sidebar.tsx

```diff
@@ -98,7 +98,7 @@ function sortSubs(a: Submission, b: Submission) {
     return 0;
   }
 
-  if (aStatus == 'in_progress') {
```

Comment:
Have you checked that there are no weird intended side effects expected with that check ?

Reply:
pretty sure this was a mistake, aStatus is `const aStatus = a.status.toLowerCase();` - so definitely a string at this point.

Reply:
makes sense 👍